### ISSUE TITLE
About Inlay hints

### ISSUE THREAD

**Issue by Nasller** [Nasller/CodeGlancePro#7]
当你看到在minimap中的底部出现大量空白时或者视窗不在正确的位置，这很有可能是Inlay hints导致的，它让编辑器多出了很多行！虽然我可以做到在minimap中补充一行，但是会影响到其他显示功能，为了补充这条线需要更改计算规则，我尝试改过，最后会导致quick glance功能计算位置错误，所以决定不更改这个错误，因为它本身不存在于编辑器中，谢谢大家。

**Comment by Nasller** [comment]
When you see a lot of white space at the bottom of the minimap, it's most likely caused by Inlay hints, which add a lot of extra lines to the editor! Although I can add a line to the minimap, it will affect other display functions. In order to supplement this line, the calculation rules need to be changed. I tried to change it, and finally it will cause the quick glance function to calculate the position error, so I decided not to change this error. Because it doesn't exist in the editor itself, thank you all.
